src/ui/public/doc_table/components/table_row.js

```diff
@@ -92,18 +102,30 @@ module.directive('kbnTableRow', function ($compile) {
           openRowHtml
         ];
 
+        const mapping = indexPattern.fields.byName;
         if (indexPattern.timeFieldName) {
           newHtmls.push(cellTemplate({
             timefield: true,
-            formatted: _displayField(row, indexPattern.timeFieldName)
+            formatted: _displayField(row, indexPattern.timeFieldName),
+            filterable: mapping[indexPattern.timeFieldName].filterable,
+            column: indexPattern.timeFieldName
           }));
         }
 
         $scope.columns.forEach(function (column) {
+          const flattenedRow = indexPattern.flattenHit(row);
+          const isFilterable =
+            flattenedRow[column] === undefined
+            ? false
+            : !mapping[column]
+            ? false
+            : mapping[column].filterable;
```

Comment:
I generally dislike custom functions where a general one will do. Now I have to find and understand `isFilterable` which helps me only in this context. If I don't know how `_.get` works, I look it up, and now I know how `_.get` works *everywhere* it's used. It's kind of a "teach a man to fish" sort of thing.

> Also, get returns the default value only for undefined, but this logic returns false if mapping[column] is falsy, so it's not quite a true replacement.

First of all, that's not true: http://codepen.io/anon/pen/EZKNNN?editors=0011
But even if it were, false would be the correct value in that situation...

> I also had to hold a lot more state in my head to map input to output (like the double negative of "not undefined").

I'd be ok with `flattenedRow[column] !== undefined` if you prefer.